robot-command/src/main/java/org/obolibrary/robot/RenameCommand.java

```diff
@@ -0,0 +1,245 @@
+package org.obolibrary.robot;
+
+import com.opencsv.CSVParserBuilder;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import java.io.*;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Options;
+import org.semanticweb.owlapi.model.OWLOntology;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** */
+public class RenameCommand implements Command {
+
+  /** Logger. */
+  private static final Logger logger = LoggerFactory.getLogger(RenameCommand.class);
+
+  /** Namespace for error messages. */
+  private static final String NS = "rename#";
+
+  /** Error message when a row does not have exactly two columns. */
+  private static final String columnCountError =
+      NS + "COLUMN COUNT ERROR line %d in file '%s' must contain exactly two values.";
+
+  /** Error message when an old IRI value is duplicated in a mappings file. */
+  private static final String duplicateMappingError =
+      NS
+          + "DUPLICATE MAPPING ERROR line %d in file '%s' contains a duplicate mapping for value '%s'.";
+
+  /** Error message when a new IRI value is duplicated and fail-on-duplicates is true. */
+  private static final String duplicateRenameError = NS + "DUPLICATE RENAME ERROR line %d in file '%s' contains a duplicate rename for value '%s'";
+
+  /** Error message when a file is not in CSV or TSV/TXT format. */
+  private static final String fileFormatError =
+      NS + "FILE FORMAT ERROR the mappings file ('%s') must be a CSV or TSV/TXT file.";
+
+  /** Error message when a mappings file does not exist. */
+  private static final String missingFileError =
+      NS + "MISSING FILE ERROR mappings file '%s' does not exist.";
+
+  /** Error message when neither a full or prefix mappings file is provided. */
+  private static final String missingMappingsError =
+      NS
+          + "MISSING MAPPINGS ERROR either a --mappings or a --prefix-mappings file must be specified.";
+
+  /** Store the command-line options for the command. */
+  private Options options;
+
+  /** Initialze the command. */
+  public RenameCommand() {
+    Options o = CommandLineHelper.getCommonOptions();
+    o.addOption("i", "input", true, "load ontology from a file");
+    o.addOption("I", "input-iri", true, "load ontology from an IRI");
+    o.addOption("o", "output", true, "save ontology to a file");
+    o.addOption("m", "mappings", true, "table of mappings for renaming");
+    o.addOption("r", "prefix-mappings", true, "table of prefix mappings for renaming");
+    o.addOption("A", "add-prefix", true, "add a new prefix to ontology file header");
+    o.addOption("f", "allow-duplicates", true, "allow two or more terms to be renamed to the same full IRI");
```

Comment:
`--allow-duplicates` is a better name, but now “f” is not a good short name. Maybe “a” or “d”, or no short name.